### DISCUSSION TITLE
Installers with multiple files

### DISCUSSION THREAD

**Discussion by Nephiaust** [microsoft/winget-pkgs#167945] [General] [answered]
Was having a read the other day, but I cant seem to find the right keywords or instructions.

I want to create a manifest that allowed an installation of a software production, but the installer is split across multiple files (e.g. install.zip; install.zip.1; install.zip.2). How do I create a manifest to do an install with that?

**Comment by Trenly** [comment, marked as answer]
This currently isn’t possible.

The closest you could get would be to add it as three separate packages and have Install-2 depend on Install-1 which itself depends on Install-0, but that would only work if each installer had a binary that can run independently (in sequence), since someone could soecify to only install Install-1 in that case. This means that the packages probably wouldn’t pass validation

**Reply by Nephiaust** [reply]
Well that sucks, and yeah the doing 3 installer packages wouldnt work as the 2 files are the splits of the original zip. So theres no "installer" or similar for it.

Cheers for answering :)

**Comment by stephengillie** [comment]
This reminds of the WInZip multi-part compressed file format, designed for storage and transfer over newsgroups and email with low file size limits. Some archives were split across dozens of `.z0n` files. But agreed, the package manager doesn't support this archive file format.

**Comment by Nephiaust** [comment]
Yeah, basically. Except the first file is the exe for self-extraction

Regards,


Nathaniel Mitchell
